### PR TITLE
Modified usage of bootstrap command

### PR DESCRIPTION
**Pull request by ShaunakJoshi1407**
- Added cluster-name in usage of bootstrap command
- Updated Readme for the same

### CODE REVIEW

**Review by nikhilbandi — DISMISSED** [review]
Looks good.

File: pkg/pmk/cluster.go [hunk 1]
```diff
@@ -43,11 +43,12 @@ func Bootstrap(ctx objects.Config, c client.Client, req qbert.ClusterCreateReque
 	s.Stop()
 
 	if err != nil {
+		fmt.Println(color.Red("x ") + " Unable to create a Cluster")
```
Comment (nikhilbandi) [inline pkg/pmk/cluster.go:46]: Nit: Unable to create cluster

File: pkg/pmk/cluster.go [hunk 2]
```diff
@@ -43,11 +43,12 @@ func Bootstrap(ctx objects.Config, c client.Client, req qbert.ClusterCreateReque
...
+		zap.S().Debug("Unable to create a Cluster. Error:", err)
```
Comment (nikhilbandi) [inline pkg/pmk/cluster.go:47]: Nit: Unable to create cluster. Error:

File: pkg/pmk/cluster.go [hunk 3]
```diff
@@ -117,13 +119,14 @@ func Bootstrap(ctx objects.Config, c client.Client, req qbert.ClusterCreateReque
...
 		fmt.Println(color.Red("x ") + " Unable to attach-node to cluster")
+		zap.S().Debug("Unable to attach-node to cluster. Error:", err)
 		if err = c.Segment.SendEvent("Attach-Node(Bootstrap)", keystoneAuth, checkFail, ""); err != nil {
 			zap.S().Errorf("Unable to send Segment event for bootstrap node. Error: %s", err.Error())
 		}
 
 		//Deleting the cluster if the node is not attached to the cluster
 		DeleteClusterBootstrap(clusterID, c, keystoneAuth, token)
-		return fmt.Errorf("Unable to attach node to the cluster"+req.Name+": %w", err)
+		return fmt.Errorf("Unable to attach node to the cluster " + req.Name)
```
Comment (nikhilbandi) [inline pkg/pmk/cluster.go:129]: Nit: Unable to attach node to cluster

File: pkg/pmk/cluster.go [hunk 4]
```diff
@@ -43,11 +43,12 @@ func Bootstrap(ctx objects.Config, c client.Client, req qbert.ClusterCreateReque
 	s.Stop()
 
 	if err != nil {
+		fmt.Println(color.Red("x ") + " Unable to create a Cluster")
+		zap.S().Debug("Unable to create a Cluster. Error:", err)
 		if err = c.Segment.SendEvent("Cluster creation(Bootstrap)", keystoneAuth, checkFail, ""); err != nil {
 			zap.S().Errorf("Unable to send Segment event for bootstrap node. Error: %s", err.Error())
 		}
-		fmt.Println(color.Red("x ") + " Unable to create a Cluster")
```
Comment (nikhilbandi) [inline pkg/pmk/cluster.go:49]: Nit: Unable to create cluster.

**Review by nikhilbandi — CHANGES_REQUESTED** [review]
Added minor comments.

File: pkg/pmk/cluster.go (hunk 3 above)
Comment (ShaunakJoshi1407) [reply pkg/pmk/cluster.go]: Done

File: pkg/pmk/cluster.go (hunk 2 above)
Comment (ShaunakJoshi1407) [reply pkg/pmk/cluster.go]: Done

File: pkg/pmk/cluster.go (hunk 1 above)
Comment (ShaunakJoshi1407) [reply pkg/pmk/cluster.go]: Done

File: pkg/pmk/cluster.go (hunk 4 above)
Comment (ShaunakJoshi1407) [reply pkg/pmk/cluster.go:49]: Resolving this as it is already addressed above

**Review by nikhilbandi — APPROVED** [review]
Changes look good.

**Pull request merged**